wasabi/printer.py

```diff
@@ -94,37 +94,38 @@ def good(
 
     def fail(
         self,
-        title: str = "",
-        text: str = "",
-        show: bool = True,
-        spaced: bool = False,
-        exits: Optional[int] = None,
+        title: Any = "",
+        text: Any = "",
+        **kwargs,
     ):
         """Print an error message.
 
-        title (str): The main text to print.
-        text (str): Optional additional text to print.
+        title (Any): The main text to print.
+        text (Any): Optional additional text to print.
         show (bool): Whether to print or not. Can be used to only output
             messages under certain condition, e.g. if --verbose flag is set.
         spaced (bool): Whether to add newlines around the output.
         exits (Optional[int]): Optional toggle to perform a system exit.
         """
+        show: bool = kwargs.get("show", True)
+        spaced: bool = kwargs.get("spaced", False)
+        exits: Optional[int] = kwargs.get("exits", None)
```

Comment:
The `str`->`Any` is fine, but I am not sure we want to do this part?

Reply:
What's the alternative?

Reply:
The alternative is to modify spacy instead. It's weird not to have these as actual kwargs.

Reply:
Yea, I agree. I presume it'll be slightly more hassle to update spaCy and sync the dependency versions, but it feels weird to remove the actual kwargs here.